tests/integration/esgf/search_results/Amon_r1i1p1_historical,rcp85_INM-CM4_CMIP5_tas.json

```diff
@@ -133,7 +133,7 @@
         "format": [
             "netCDF, CF-1.4"
         ],
-        "index_node": "esgf-index1.ceda.ac.uk",
+        "index_node": "esgf.ceda.ac.uk",
```

Comment:
Is it necessary to update the index node in these old search results used for testing?

Reply:
If you think that's best, it's fine with me.